### ISSUE TITLE
Genie python: Plot spectrum error

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#5562]
During the system testing it was found that `g.plot_spectrum(1)` failed with the following error:
```
WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/jquery-ui.min.css (127.0.0.1) 0.00msWARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/external/jquery/jquery.js (127.0.0.1) 0.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/jquery-ui.min.js (127.0.0.1) 1.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/jquery-ui.min.css (127.0.0.1) 1.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/jquery-ui.min.js (127.0.0.1) 1.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/external/jquery/jquery.js (127.0.0.1) 1.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/jquery-ui.min.css (127.0.0.1) 1.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/jquery-ui.min.js (127.0.0.1) 1.00ms   WARNING:tornado.access:404 GET /_static/jquery-ui-1.12.1/external/jquery/jquery.js (127.0.0.1) 1.00ms
```

This must be fixed before release.

subticket of #5441

**Comment by Tom-Willemsen** [comment]
https://github.com/ISISComputingGroup/genie_python/pull/277

**Comment by John-Holt-Tessella** [comment]
Merged on 7.1.0
